libraries/AP_AHRS/AP_AHRS_View.h

```diff
@@ -118,12 +118,12 @@ class AP_AHRS_View
         return ahrs.get_relative_position_NE_origin(vecNE);
     }
     
-    bool get_relative_position_D_home(float &posD) const {
-        return get_relative_position_D_home(posD);
+    void get_relative_position_D_home(float &posD) const {
+        return ahrs.get_relative_position_D_home(posD);
```

Comment:
I don't really like to do a return from a void method, but this is a bit nitpicking. Other than this looks good.